apps/server/src/modules/board/uc/element.uc.spec.ts

```diff
@@ -281,4 +281,36 @@ describe(ElementUc.name, () => {
 			});
 		});
 	});
+
+	describe('checkElementReadPermission', () => {
+		const setup = () => {
+			const user = userFactory.build();
+			const drawingElement = drawingElementFactory.build();
+
+			return { drawingElement, user };
+		};
+
+		it('should execute properly', async () => {
```

Comment:
this is only testing that find by id is called, but nothing that the permissions check is also called
so, you can rename this test accordingly, and write a new test to check the permission...